packages/algorand-js/src/runtime/runtime.ts

```diff
@@ -98,14 +98,24 @@ export class Runtime {
     return undefined;
   }
 
+  /**
+   * Fetches local state for account address and application index
+   * @param appId application index
+   * @param accountAddr address for which local state needs to be retrieved
+   */
+  getLocalState (appId: number, accountAddr: string, key: Uint8Array): StackElem | undefined {
```

Comment:
Please add one more param `key` in description

Reply:
ok